tests/components/climate/test_generic_thermostat.py

```diff
@@ -54,13 +54,16 @@ def test_setup_missing_conf(self):
                 'climate': config})
 
     def test_valid_conf(self):
-        """Test set up genreic_thermostat with valid config values."""
-        self.assertTrue(setup_component(self.hass, 'climate',
-                        {'climate': {
-                            'platform': 'generic_thermostat',
-                            'name': 'test',
-                            'heater': ENT_SWITCH,
-                            'target_sensor': ENT_SENSOR}}))
+        """Test set up genric_thermostat with valid config values."""
```

Comment:
Different typo than in the original, but still a typo. ;-)

Reply:
😉 sure, you are right. Too fast last-moment change 😜 I will fix better tomorrow